frontend/components/product-list/sections/FilterableProductsSection/ProductGrid.tsx

```diff
@@ -20,7 +20,6 @@ export function ProductGrid({ children }: ProductGridProps) {
    return (
       <SimpleGrid
          data-testid="grid-view-products"
-         bg="gray.100"
```

Comment:
Do we still want to remove these styles?

Reply:
I believe these styles would still result in empty tiles on the last page if towards the end the # of products is < 18.
That's why I decided to leave the changes where I delete this. 
For example, if I add the `bg="gray.100"` we would end up with the last page looking like:
<img width="497" alt="Screenshot 2022-07-13 142511" src="https://user-images.githubusercontent.com/50181909/178839079-23b839e6-0091-4ab2-9d7c-ad646e4c4e7d.png">

Reply:
Unless there is a way to fix that as well, I think keeping the commit where `bg="gray.100"` is deleted would fix this last page issue.

Reply:
un_deploy_block 🦋